lib/instance_billing_flavor_check/utils.py

```diff
@@ -25,11 +25,9 @@
 from lxml import etree
 
 logger = logging.getLogger(__name__)
+log_filename = '/var/log/{}'.format(__name__.split('.')[0])
```

Comment:
but `__name__ is `utils.py` we do not want alogfile names `utils` we'd want `instance_flavor_check.log`

Reply:
`__name__` is `instace_flavor_check.utils.py` then `split('.')` would return `['instance_flavor_check', 'utils.py']`
and we take the first element